INSTALL.md

```diff
@@ -103,3 +111,24 @@ make py-deps
 ```
 
 See [GitHub Actions workflow](.github/workflows/main.yml) for other usages of the Makefile.
+
+## Tests
+
+To test the WebAssembly module we will use the [cypress](https://www.cypress.io/) test framework.
+
+We want to test if visiting the example web page renders the answer `-1.00`.
+
+```{.js file=cypress/integration/example_spec.js}
+describe('src/js/example.html', () => {
+  it('should render -1.00', () => {
+    cy.visit('http://localhost:8000/src/js/example.html');
+    cy.get('#answer').contains('-1.00');
+  });
+});
+```
+
+The test can be run with
+
+```{.awk #test-wasm}
+npx cypress run --config-file false
```

Comment:
add short instructions for npx installation.

Reply:
Sometimes build utils (make etc.) are not installed by default. Can you add this to dependency list as well.

Reply:
Done in 03fd466f3e6e159811ef8d12c320cec709f16f53 and b7227b01ed149a1f9ef2567153a88b4e357e94da